.env.tpl

```diff
@@ -38,7 +38,7 @@ LDAP_URL=op://Dimpact/ZAC-.env-$APP_ENV/LDAP/URL
 LDAP_USER=op://Dimpact/ZAC-.env-$APP_ENV/LDAP/USER
 MAILJET_API_KEY=op://Dimpact/ZAC-.env-$APP_ENV/MAILJET/API_KEY
 MAILJET_API_SECRET_KEY=op://Dimpact/ZAC-.env-$APP_ENV/MAILJET/API_SECRET_KEY
-MAX_FILE_SIZE_MB=op://Dimpact/ZAC-.env-$APP_ENV/MAX_FILE_SIZE_MB
+MAX_FILE_SIZE_MB=80
```

Comment:
hard-coded op 80 MB? waarom niet via die `MAX_FILE_SIZE_MB` env?

Reply:
Dit _is_ die MAX_FILE_SIZE_MB env var. Wat bedoel je @bas-info-nl ? Dit is juist goed want uit 1Password halen heeft geen toegevoegde waarde.

Reply:
Ah - verkeerd gelezen - thx 👍